### ISSUE TITLE
Issue with OBS Scene.

### ISSUE THREAD

**Issue by XCanG** [WatchBeam/ftl-studio#49]
First of all I would ask: what OBS it use as base? Classic OBS or OBS Studio (previously named as OBS Multiplatform)?

If it use OBS Studio, I have bug with scene, what rendered in OBS Studio right.

**Comment by Maximilian-Reuter** [comment]
yes it uses OBS studio but most of these issues are gonna disappear once they release their OBS plugin
which will let you use the latest obs studio. Should happen at the end of September according to their latest email.

**Comment by XCanG** [comment]
Hm, ok. Also, didn't see any difference with FTL mode or RTMP, but just see, that FTL encoder work more busy. For example if I stream in OBS 1080p60 normally, on FTL encoding overflow and I may stream only 1080p30 or on stream it will laggy. Also both streams delaying is 3 seconds.

**Comment by Maximilian-Reuter** [comment]
If you don't see a difference in delay with ftl, then you are doing something wrong.

Here is my version of OBS with the ftl plugin:https://mega.nz/#!K85gkBYJ!-QWDjNN-9f2O1sOcUpZXkANJ7meaEhfcxijhb_WTxhA

Simply change in settings under "Stream" the stream type to "Beam FTL Service" to use FTL

**Comment by ProbablePrime** [comment]
For experiment sake, try a lower bitrate with tachyon. You'll see significant delay reduction. 

Could you describe your issue with your scene?
